cupy_backends/cuda/libs/cusparse.pyx

```diff
@@ -4835,6 +4857,7 @@ cpdef size_t sparseToDense_bufferSize(intptr_t handle, size_t matA,
 
 cpdef sparseToDense(intptr_t handle, size_t matA, size_t matB, int alg,
                     intptr_t buffer):
+    _setStream(handle)
```

Comment:
Is this a fix for cuSPARSE too?

Reply:
Yes I think so! We will need to backport this.

Reply:
can we do it in a separate PR?, but I am not sure if we do it hipSPARSE will be broken

If you separate it I can merge within today

Reply:
Sure I will send another PR. But I am still fighting with one last big file `test_linalg.py`, so probably not today...

Reply:
See #5507.